src/generic/gridctrl.cpp

```diff
@@ -16,8 +16,7 @@
 
 #if wxUSE_GRID
 
-#include "wx/generic/gridctrl.h"
-#include "wx/generic/grideditors.h"
+#include "wx/grid.h"
```

Comment:
Also FYI: I'm not applying this hunk because I don't think it's really needed -- but please let me know if I'm missing something here.